### PR TITLE
[WX-766] Remove dsub

### PR DESCRIPTION
**Pull request by breilly2**
Remove code related to dsub
Update docs to reflect removal of dsub support

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,9 +83,6 @@ Note that a “task” in dsub nomenclature corresponds to a Job Manager API’s
 
 ### Server Setup
```
Comment (kraefrei) [inline README.md:84]: Minor detail, but do we want to consolidate this section?

File: common-compose.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 version: '2.1'
 services:
   rebuild-swagger:
-    image: java
+    image: eclipse-temurin:latest
```
Comment (kraefrei) [inline common-compose.yml:4]: I think this may default to latest without specifying. Doesn't hurt to be extra clear though

**Review by kraefrei — APPROVED** [review]
LGTM

**Review by aednichols — APPROVED** [review]
LGTM, thanks! 👍

**Pull request merged**